src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -422,11 +446,17 @@ public boolean isCorrectMD5(final String otherMD5)
     /**
      * Reads a template from an inputstream.
      */
-    private static Template readTemplateFromStream(final InputStream stream) throws IOException
+    private static Template readTemplateFromStream(final InputStream stream, final DataFixer fixer) throws IOException
     {
         final NBTTagCompound nbttagcompound = CompressedStreamTools.readCompressed(stream);
+
+        if (!nbttagcompound.hasKey("DataVersion", 99))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 99 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)